docs/maintenance.rst

```diff
@@ -44,6 +44,34 @@ After updating the ``apexsh.dat`` file, some of the unit tests and the
 documentation examples in the README and ``apexpy/docs/examples`` directory
 will need to be updated as well.
 
+Modifying Fortran Source
+------------------------
+When modifying the fortran source code, it can be helpful to run a preliminary
+validation of the fortran output independent of the python wrapper.
+
+1. Remove any existing binaries by running the ``make clean`` command.
+2. Build the ``apextest`` binary by running the ``make`` command.
+3. Execute the ``apextext`` binary.
+4. Confirm the output printed to the screen matches the test output shown in
+   the comment blot at the bottom of ``checkapexsh.f90``.
```

Comment:
```suggestion
1. Remove any existing binaries by running the ``make clean`` command.
2. Build the ``apextest`` binary by running the ``make`` command.
3. Copy the IGRF coefficient file (``cp ../apexpy/igrf13coeff.txt``) into the current directory.
4. Execute the ``apextext`` binary.
5. Confirm the output printed to the screen matches the test output shown in
   the comment blot at the bottom of ``checkapexsh.f90``.
```

Reply:
Changed this and added the same step to the earlier section on updating ``apexsh.dat``.